ReactComponents/UserView.js

```diff
@@ -71,6 +75,11 @@ var UserView = React.createClass({
   handleUserActivityEvent: function(campaignActivity) {
     this.fetchData();
   },
+  handleUserAvatarEvent: function(response) {
+    this.setState({
+      photo: response.photo + '?time=' + Date.now(),
```

Comment:
Workaround for cached avatar (refs https://github.com/DoSomething/northstar/issues/272#issuecomment-189073894)